cli/config/credentials/native_store.go

```diff
@@ -76,6 +77,7 @@ func (c *nativeStore) GetAll() (map[string]types.AuthConfig, error) {
 		ac.Username = creds.Username
 		ac.Password = creds.Password
 		ac.IdentityToken = creds.IdentityToken
+		ac.ServerAddress = creds.ServerAddress
```

Comment:
is `ac.ServerAddress` ever a non-empty value?

Reply:
I do not believe so no. Since in both stores the code always sets it to what the Registry Hostname is. Is your idea to just have creds be returned there directly?

Reply:
@dmcgowan do you mean you prefer to make this conditional to keep that option open?

Something like this?;

```go
if ac.ServerAddress  == "" {
    ac.ServerAddress = creds.ServerAddress
}
```

Reply:
I had not read it that way yet. That makes sense. So we only set it when it is empty and not accidentally overwrite a potential value. I can add that real quick.